roles/ceph-rgw/tasks/rgw_create_pools.yml

```diff
@@ -0,0 +1,67 @@
+---
+  - name: remove ec profile
+    command: "{{ container_exec_cmd }} ceph --connect-timeout 5 --cluster {{ cluster }} osd erasure-code-profile rm {{ item.value.ec_profile }}"
+    with_dict: "{{ rgw_create_pools }}"
+    delegate_to: "{{ groups[mon_group_name][0] }}"
+    changed_when: false
+    run_once: true
+    when: 
+      - item.value.pool_type is defined  
+      - item.value.pool_type == 'ec'
+    ignore_errors: yes
+
+  - name: set ec profile
+    command: "{{ container_exec_cmd }} ceph --connect-timeout 5 --cluster {{ cluster }} osd erasure-code-profile set {{ item.value.ec_profile }} k={{ item.value.ec_k }} m={{ item.value.ec_m }}"
+    with_dict: "{{ rgw_create_pools }}"
+    delegate_to: "{{ groups[mon_group_name][0] }}"
+    changed_when: false
+    run_once: true
+    when: 
+      - item.value.pool_type is defined  
+      - item.value.pool_type == 'ec'
+
+  - name: set crush rule
+    command: "{{ container_exec_cmd }} ceph --connect-timeout 5 --cluster {{ cluster }} osd crush rule create-erasure {{ item.key }} {{ item.value.ec_profile }}"
```

Comment:
what about this command?
If the playbook is run again, will it return 0 even though it's already created? Otherwise, we might need a `failed_when: false` here as well

Reply:
@alimaredia ?